hedyorg/hedy
# 🪲 Check for programs with empty string as adventure id

Some old programs present the challenge of having an empty string in their `adventure_name` field. Luckily, this doesn't seem to happen anymore, as I checked the DB and in production only programs from around 2022 present this issue. 

**How to test**

* Copy and paste an existing program in the DB, change the `id` and put an emtpy string in the `adventure_name` field. On main, it should crash, using this branch, the programs page should be shown normally. 

Here's an example for the user `student1`:

```json
    {
      "session": "fac24066d19f4fd8ac44508ac4a7fdae",
      "date": 1709241804445,
      "lang": "en",
      "version": "DEV",
      "level": 1,
      "code": "ask What's your name?\necho so your name is\nask What's your name?\necho so your name isask What's your name?\necho so your name isask What's your name?\necho so your name isask What's your name?\necho so your name isask What's your name?\necho so your name is",
      "name": "adventure1 test test",
      "username": "student1",
      "adventure_name": "",
      "id": "ididididididididididi",
      "username_level": "student1-1",
      "public_profile": null
    }
```
